mozci/task.py

```diff
@@ -358,6 +360,30 @@ def configuration(self):
         parts = config.split("-")
         return "-".join(parts[:-1] if parts[-1].isdigit() else parts)
 
+    @property
+    def overhead(self):
```

Comment:
This is defined on a specific task, but we will have to define `overheads`, `total_overhead`, `median_overhead` on LabelSummary to actually use the information (e.g. the median overhead for a given suite on a given platform/configuration).

For task-level scheduling, we can just consider the sum of the median durations of all scheduled tasks.
For manifest-level scheduling, for each task we'll have to sum the durations of the groups in the task and sum the overhead associated with the task (I think by having a median overhead by suite and platform/configuration, we might be precise enough; maybe even just platform/configuration?).

Reply:
Right, I will implement them while I wait for review from ahal and ekyle.

Reply:
@worldomonation I will require the group-level run times, but I plan to compute the aggregates after the data has been put into the database.  Specifically, there is a BigQuery database with some of the data generated by `mozci`; my plan is to use include what these functions generate and form queries from there.